notebook/notebookapp.py

```diff
@@ -704,6 +708,44 @@ class NotebookApp(JupyterApp):
 
     _log_formatter_cls = LogFormatter
 
+    _json_logging_import_error_logged = False
+
+    log_json = Bool(False, config=True,
```

Comment:
Should I write up a small guide in the docs about using this, somewhere like here?

https://github.com/jupyter/notebook/blob/master/docs/source/config_overview.rst#notebook-server

Or is the config option help sufficient?

Reply:
I think the help text you've provided is sufficient.  The other options are explicitly listed that I can see, and I don't think everyday users will be wanting JSON logging.  This is more of a large-scale installation thing.